runtime-decompiler/src/main/java/org/jrd/backend/communication/FsAgent.java

```diff
@@ -46,17 +47,18 @@ public FsAgent(List<File> cp) {
     public String submitRequest(final String request) {
         String[] q = request.split("\\s+");
         try {
-            switch (q[0]) {
-                case "CLASSES":
+            switch (AgentRequestAction.RequestAction.fromString(q[0])) {
```

Comment:
I am not following how this change relates to the commit message of "Made INIT_CLASS in fsagent non fatal" - the fromString method is the only change in that commit.

Reply:
Yah. Al this work my beter self was suffering from hard-coded "REPLICATED_MEMBER_OF_ENUM". In this last commit I could not bare that.
All hardcoded occurences should be fixed similarly.